### PR TITLE
Tests for interfaces: Tree, TreeExpansionService

### PR DESCRIPTION
**Pull request by theBadT**
This change contributes to: https://github.com/theia-ide/theia/issues/448
I've written tests for Tree and TreeExpansionService API

### CODE REVIEW

**Comment by theBadT** [comment]
tests are failing. I can't see in how far my change might have caused this

**Comment by vince-fugnitto** [comment]
@theBadT thank you for your contribution! 😄 
Before we can accept you changes the following must be completed.

Please make sure you have signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php) as soon as possible.

We also must sign off our commits (that is why the `DCO` check fails).
I also noticed you picked up unnecessary commits in your PR (currently 10), can you please address your history?

File: packages/core/src/browser/tree/tree-expansion.spec.ts [hunk 1]
```diff
@@ -0,0 +1,158 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```
Comment (akosyakov) [inline packages/core/src/browser/tree/tree-expansion.spec.ts]: 2019
your org or name

**Comment by akosyakov** [comment]
@theBadT thanks for doing it ❤️

**Comment by theBadT** [comment]
basically I'm not able to squash the commits during a rebase.
GIT_TRACE=1 git rebase --interactive origin/master~9 master results in
gpg: cannot open tty `no tty': No such file or directory
gpg (GnuPG) 1.4.22
any suggestions?

**Comment by akosyakov** [comment]
@theBadT We don't care about GPG signing, so you can disable it while contributing to Theia: https://stackoverflow.com/questions/39274739/how-to-disable-git-gpg-signing

**Pull request closed without merging**